module-system/module-implementations/sov-evm/src/evm/test_helpers.rs

```diff
@@ -0,0 +1,39 @@
+use bytes::Bytes;
+use ethers_contract::BaseContract;
+use ethers_core::abi::Abi;
+use revm::primitives::{ExecutionResult, Output, B160};
+use std::path::PathBuf;
+
+pub(crate) fn output(result: ExecutionResult) -> Bytes {
+    match result {
+        ExecutionResult::Success { output, .. } => match output {
+            Output::Call(out) => out,
+            Output::Create(out, _) => out,
+        },
+        _ => panic!("Expected successful ExecutionResult"),
```

Comment:
Is it ok to panic here? Maybe use `TryFrom` ?

Reply:
oh, it is foreign type, but maybe we can return result

Reply:
It is for tests, nevermind

Reply:
This is test helper and in testes I prefer to panic early.